aplus/settings.py

```diff
@@ -158,6 +158,11 @@
     'aplus_auth',
 )
 
+# Sitemap settings
+##########################################################################
+BASE_PRIORITY = 0.5
```

Comment:
Perhaps add a short comment on both variables, about what they stand for? So that system admin does not need to go reading to code to figure it out.

Reply:
Done!